packages/console/src/pages/TenantSettings/Subscription/DowngradeConfirmModalContent/PlanQuotaDiffList/QuotaDiffItem/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { conditional } from '@silverhand/essentials';
+import classNames from 'classnames';
+
+import DescendArrow from '@/assets/icons/descend-arrow.svg';
+import Failed from '@/assets/icons/failed.svg';
+import DynamicT from '@/ds-components/DynamicT';
+import {
+  quotaItemUnlimitedPhrasesMap,
+  quotaItemPhrasesMap,
+  quotaItemLimitedPhrasesMap,
+} from '@/pages/TenantSettings/Subscription/quota-item-phrases';
+import { type SubscriptionPlanQuota } from '@/types/subscriptions';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  isChangeStateVisible?: boolean;
+  quotaKey: keyof SubscriptionPlanQuota;
+  quotaValue: SubscriptionPlanQuota[keyof SubscriptionPlanQuota];
+};
+
+function QuotaDiffItem({ isChangeStateVisible = false, quotaKey, quotaValue }: Props) {
+  const isUnlimited = quotaValue === null;
+  const isNotCapable = quotaValue === 0 || quotaValue === false;
+  const isLimited = Boolean(quotaValue);
+
+  const Icon = isNotCapable ? Failed : DescendArrow;
+
+  return (
+    <li className={classNames(styles.item, isChangeStateVisible && styles.withChangeState)}>
```

Comment:
Can we change the name to `hasIcon && styles.hasIcon`? This is easier to understand